apps/spruce/src/pages/version/TestAnalysis/GroupedTestMapList/FailedTestGroup/index.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import { Body } from "@leafygreen-ui/typography";
+import pluralize from "pluralize";
+import TaskStatusBadge from "@evg-ui/lib/components/Badge/TaskStatusBadge";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { Accordion } from "components/Accordion";
+import { TaskBuildVariantField } from "../../types";
+import FailedTestGroupTable from "./FailedTestGroupTable";
+
+interface FailedTestGroupProps {
+  testName: string;
+  tasks: TaskBuildVariantField[];
+}
+
+const FailedTestGroup: React.FC<FailedTestGroupProps> = ({
+  tasks,
+  testName,
+}) => {
+  const statusMap = countStatuses(tasks);
+  const sortedStatuses = Object.entries(statusMap).sort((a, b) =>
+    a[0].localeCompare(b[0]),
+  );
+  return (
+    <Accordion
+      shouldRenderChildIfHidden={false}
```

Comment:
Could you explain about what this prop does? I couldn't really tell after toggling it off